src/org/zaproxy/zap/extension/api/ApiImplementor.java

```diff
@@ -321,7 +328,25 @@ public ApiResponse handleApiAction(String name, JSONObject params) throws ApiExc
 	public HttpMessage handleApiOther(HttpMessage msg, String name, JSONObject params) throws ApiException {
 		throw new ApiException(ApiException.Type.BAD_OTHER, name);
 	}
-	
+
+	/**
+	 * Override if implementing one or more 'persistent connection' operations. 
+	 * These are operations that maintain long running connections, potentially staying alive
+	 * as long as the client holds them open.
+	 * @param msg the HTTP message containing the API request
+	 * @param httpIn the input stream
+	 * @param httpOut the output stream
+	 * @param name the name of the requested pconn endpoint
+	 * @param params the API request parameters
+	 * @return
+	 * @throws ApiException if an error occurred while handling the API pconn endpoint
+	 */
+	public boolean handleApiPersistentConnection(HttpMessage msg,
```

Comment:
This could be changed to return `void` now.

Reply:
Done :)